epub33/core/index.html

```diff
@@ -1382,6 +1382,25 @@ <h5>HTML <code>img</code> fallbacks</h5>
 									href="#sec-manifest-fallbacks">manifest fallback</a>.</li>
 						</ul>
 					</section>
+					<section>
+						<h5>HTML <code>script</code> element</h5>
+
+						<p>Although the [[HTML]] [^script^] element lacks intrinsic fallback capabilities, it is not
+							required that EPUB creators provide fallbacks for <a data-cite="html#data-block">data
+								blocks</a> [[HTML]]. As the <code>script</code> element does not represent user content,
+							data blocks are not rendered unless manipulated by script.</p>
+
+						<div class="note">
+							<p>The exemption from fallbacks aligns data blocks with the exemption made to <a
+									href="confreq-foreign-no-fallback">allow data files</a> to travel in the [=EPUB
+								container=]. As data blocks are not separate resources from their host [=EPUB content
+								document=], they do not fall under the same exemption.</p>
+						</div>
+						<div class="note">
+							<p>Since the [[SVG]] <span data-cite="svg">[^script^] element</span> is an implementation of
+								the HTML element, the same exemption applies to both formats.</p>
+						</div>
```

Comment:
Tiny comment. In HTML, the `href` attribute in a `script` element used for datablock is ignored. Ie, a non-JS content is _always_ embedded. This explicit statement is missing in the SVG version of the `script` element's definition; it is therefore, in theory, possible to say something like

```xml
<script href="http://example.org/a.ttl" type="text/turtle"/>
```

I regard that as a bug in the SVG spec...

I think we can safely ignore this dark corner in the spec, I just want to do note this to ourselves...

Reply:
Ya, this is why I mention SVG `script` in a note rather than cite both elements as distinct. SVG doesn't mention data blocks at all, and seems to assume that all `script` elements contain some kind of scripting and only to not execute ones with unknown media types.

It just jogged my memory that we ran into this earlier while defining scripted content documents. In that case, we put in this note:

> [[svg](https://w3c.github.io/epub-specs/epub33/core/#bib-svg)] does not define data blocks as of publication, but the same exclusion would apply if a future update adds the concept.

https://w3c.github.io/epub-specs/epub33/core/#sec-scripted-support

Maybe for consistency we should use the same note here.

Reply:
Your memory is bottomless... 😀

Yes, the same remark should apply here.